Add new db commands

allanchau opened the pull request:
This PR adds some db scripts to the cli.

## Setup

Add something like this to your c.js file
```javascript
mongo: {
        beta: {
            host: 'some url',
            name: 'aesso-beta',
            password: '',
            port: '12345',
            user: 'aesso',
        },
        local: {
            host: 'db.common.idearium.local',
            name: 'aesso',
            port: '12345',
            user: 'aesso',
        },
        production: {
            host: 'some url',
            name: 'aess-beta',
            password: 'password',
            port: '12345',
            user: 'aesso',
        },
    },
```

## Testing

- [x] Run `c mongo` to see a list of the available commands.
- [x] Run through all the commands and test.

smebberson commented on bin/c-db-import.js:
```diff
@@ -0,0 +1,37 @@
+}
+
+Promise.all([
+    loadConfig(`mongodb.${env}`),
```
@allanchau, I think it would be better to just execute `loadConfig('mongodb')` once, and then grab the environments yourself. More performant.

smebberson commented on bin/c.js:
```diff
@@ -8,6 +8,7 @@ const { missingCommand } = require('./lib/c');
 // The basic program, which uses sub-commands.
 program
     .command('d <command>', 'Shortcuts to control Docker.')
+    .command('db <command>', 'Shortcuts to help with the database.')
```
Hmm, should this be mongo? What if we wanted to do the same for Redis or MySQL or some other database?

allanchau commented:
@smebberson I've updated this to use `mongo` in `c.js` now and changed the command to `c mongo ...`.
Also added some fixes for `c project init` and updated it to ask if mongo is used.
